### PR TITLE
Prevent non geocodable addresses for adherent and committees

### PR DESCRIPTION
**Pull request by hhamon**

### CODE REVIEW

File: src/AppBundle/Address/AddressInterface.php [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+     * Returns the french city representation (zip code + insee code).
+     *
+     * For example: 92110-92024
+     *
+     * @return string|null
+     */
+    public function getCity();
+
+    /**
+     * Returns the 2 char country code.
```
Comment (HeahDude) [inline src/AppBundle/Address/AddressInterface.php]: chars

File: src/AppBundle/Address/AddressInterface.php [hunk 2]
```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace AppBundle\Address;
+
+interface AddressInterface
+{
+    /**
+     * Returns the street name.
+     *
+     * @return string
```
Comment (HeahDude) [inline src/AppBundle/Address/AddressInterface.php]: `|null`?

File: src/AppBundle/Geocoder/GeoPointInterface.php [hunk 3]
```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace AppBundle\Geocoder;
+
+interface GeoPointInterface extends GeocodableInterface
+{
+    /**
+     * Updates the coordinates of the geocodable object thanks to the data
+     * stored in the Coordinates value object.
```
Comment (HeahDude) [inline src/AppBundle/Geocoder/GeoPointInterface.php]: `{@link Coordinates}`?

File: tests/AppBundle/Controller/AdherentControllerTest.php [hunk 4]
```diff
@@ -105,13 +105,14 @@ public function testEditProfileFillAdherentData()
...
         $this->assertResponseStatusCode(Response::HTTP_OK, $this->client->getResponse());
-        $this->assertSame(6, $errors->count());
-        $this->assertSame('Cette valeur ne doit pas être vide.', $errors->eq(0)->text());
-        $this->assertSame('Cette valeur ne doit pas être vide.', $errors->eq(1)->text());
-        $this->assertSame('Cette ville et ce code postal ne sont pas liés.', $errors->eq(2)->text());
-        $this->assertSame("Cette valeur n'est pas un identifiant valide de ville française.", $errors->eq(3)->text());
-        $this->assertSame("L'adresse est obligatoire.", $errors->eq(4)->text());
-        $this->assertSame("Cette valeur n'est pas valide.", $errors->eq(5)->text());
+        $this->assertSame(7, $errors->count());
+        $this->assertSame('Cette valeur ne doit pas être vide.', $errors->eq(0)->text(), 'First name should be erroneous.');
```
Comment (HeahDude) [inline tests/AppBundle/Controller/AdherentControllerTest.php]: Sorry after rebasing, you should have removed the third argument of `assertSame`, my bad!

File: tests/AppBundle/Validator/GeocodableAddressValidatorTest.php [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+     */
+    public function testUnsupportedConstraint()
+    {
+        $this->validator->validate(
+            PostAddress::createFrenchAddress('92 bld Victor Hugo', '92110-92024'),
+            new NotBlank()
+        );
+    }
+
+    public function testSkipValidation()
```
Comment (HeahDude) [inline tests/AppBundle/Validator/GeocodableAddressValidatorTest.php]: Check but I'm surprised that's not covered by the core abstract tests.

Comment (hhamon) [inline tests/AppBundle/Validator/GeocodableAddressValidatorTest.php]: It's not already covered by the abstract test class.

Comment (HeahDude) [inline tests/AppBundle/Validator/GeocodableAddressValidatorTest.php]: Ok

**Pull request merged**